### PR TITLE
Bestpeers_empoyees_portal

### PR DESCRIPTION
**Pull request by chandraprakash9589**
1. User can  Login   in the portal 
2.User can send the status and view the status
3..User can Logout

### CODE REVIEW

File: frontend/src/components/Attendance/MyDailyStatus.jsx [hunk 1]
```diff
@@ -1,89 +1,80 @@
...
+export const MyDailyStatus = ({}) => {
   const [data, setData] = useState([]);
-  const [clicked, setIsClicked] = useState(false);
-  const [newdata,setNewdata]=useState("hello")
-
   const navigate = useNavigate();
-const token =localStorage.getItem("jwtToken");
-console.log(token, '=================P')
+  const token = localStorage.getItem("jwtToken");
+  // console.log(token, '=================P')
```
Comment (developerabhijeet) [inline frontend/src/components/Attendance/MyDailyStatus.jsx:10]: Remove console.logs

File: frontend/src/components/Attendance/MyDailyStatus.jsx [hunk 2]
```diff
@@ -1,89 +1,80 @@
...
-        const response = await axios.get("http://localhost:4500/get_daily_status", { headers: {"Authorization" : `Bearer ${token}`} }).then((res) => setData(res.data.data));;
-
+        const response = await axios
+          .get("http://localhost:4500/tasks", {
+            headers: { Authorization: `Bearer ${token}` },
+          })
+          .then((res) => setData(res.data));
       } catch (error) {
-        console.error({ error: "data is not found" });
+        console.error({ error: "data is not fouuuuuuuund" });
```
Comment (developerabhijeet) [inline frontend/src/components/Attendance/MyDailyStatus.jsx:20]: show alerts instead logging the  errors

File: frontend/src/components/Attendance/MyDailyStatus.jsx [hunk 3]
```diff
@@ -1,89 +1,80 @@
...
-            })
-          ) : (
-            null
-          )}
-
+          {data && data.length > 0
+            ? data.map((item, index) => {
+                return (
+                  <tr key={index}>
+                    {console.log(data, "data++++++=====>>")}
```
Comment (developerabhijeet) [inline frontend/src/components/Attendance/MyDailyStatus.jsx:56]: Remove console.logs

File: frontend/src/components/Attendance/SendMyDailyStatus.jsx [hunk 4]
```diff
@@ -3,62 +3,65 @@ import DatePicker from "react-datepicker";
...
+    const usersEmail = selectedOptions.map((e) => e.value);
+    setEmailData(usersEmail);
+  };
+
+  const handleStatusSubmit = async (e) => {
+    const token =localStorage.getItem("jwtToken");
+    e.preventDefault();
+    try {
+      await axios.post(
+        "http://localhost:4500/tasks",
```
Comment (developerabhijeet) [inline frontend/src/components/Attendance/SendMyDailyStatus.jsx:45]: for http://localhost:4500/ create a separate constant in utils folder like export const BaseURL = http://localhost:4500/

File: backend/src/tests/taskController.test.ts [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    expect(response.body.title).toBe(taskData.title);
+  });
+  
+  
+  it('should retrieve all tasks', async () => {
+    const tasks = [
+      { title: 'Task 1', description: 'Description 1', dueDate: new Date(), completed: false },
+      { title: 'Task 2', description: 'Description 2', dueDate: new Date(), completed: true },
+      { title: 'Task 1', description: 'Description 1', dueDate: new Date(), completed: false },
+      { title: 'Task 2', description: 'Description 2', dueDate: new Date(), completed: true },
```
Comment (developerabhijeet) [inline backend/src/tests/taskController.test.ts:45]: @chandraprakash9589 why we added constants here???

**Review by developerabhijeet — CHANGES_REQUESTED** [review]
Resolve all the comments

File: frontend/src/components/Attendance/MyDailyStatus.jsx (hunk 3 above)
Comment (chandraprakash9589) [reply frontend/src/components/Attendance/MyDailyStatus.jsx:56]: sure sir

**Review by developerabhijeet — APPROVED** [review]

**Pull request merged**